python/hsfs/core/statistics_engine.py

```diff
@@ -75,6 +64,30 @@ def compute_statistics(
             # Hive engine
             engine.get_instance().profile(metadata_instance)
 
+    @staticmethod
+    def profile_statistics(metadata_instance, feature_dataframe):
+        if len(feature_dataframe.head(1)) == 0:
+            raise exceptions.FeatureStoreException(
+                "There is no data in the entity that you are trying to compute "
+                "statistics for. A possible cause might be that you inserted only data "
+                "to the online storage of a feature group."
+            )
+        content_str = engine.get_instance().profile(
+            feature_dataframe,
+            metadata_instance.statistics_config.columns,
+            metadata_instance.statistics_config.correlations,
+            metadata_instance.statistics_config.histograms,
+        )
+        return content_str
+
+    def register_split_statistics(self, td_metadata_instance, statistics_of_splits):
+        commit_time = int(float(datetime.datetime.now().timestamp()) * 1000)
+        stats = statistics.Statistics(
+            commit_time=commit_time, content=None, split_statistics=statistics_of_splits
```

Comment:
you don't need `content=None`, it's already the default value.

Reply:
Same here -  when https://github.com/logicalclocks/feature-store-api/pull/407 gets merged, we have to also pass the `metadata_instance.statistics_config.exactUniqueness` to the `profile(...)` call.